packages/gatsby-theme-carbon/src/components/Grid.js

```diff
@@ -49,6 +49,7 @@ export const Column = ({
   noGutterLg,
   noGutterXl,
   noGutterMax,
+  noGutterSmYesGutterLg,
```

Comment:
hmm, this seems a little spicy. Does we could have thousands of combinations if this pattern continued.

I assume adding the `no-gutter-sm` impact the larger breakpoints?

Reply:
Yeppers. I couldn't figure out how to effect the column without introducing this.